### ISSUE TITLE
[BUG] Few panes in single page - pane styles bug

### ISSUE THREAD

**Issue by spb-web** [roman-rr/cupertino-pane#133]
We can try using inline styles. Then, when initializing a new pane, new "style" tags will not be created. And to apply the theme, we can use css variables. I would do it if it fits into the development plans.

**Comment by roman-rr** [comment]
@spb-web Big thanks for actual issue! 

> Then, when initializing a new pane, new "style" tags will not be created.
This is true, and i looking for best way that we can use `styles`. 

> We can try using inline styles
Good to not using inline at all. Basic styles which users could configure must be internal. 
But we can use inline for service adjustments such as inversed styles.

> And to apply the theme, we can use css variables
We can use some variables.

Feel free to create pull, i will review and adjust if needs.

**Comment by roman-rr** [comment]
@spb-web I think we can just apply additional classes for configurations. 

E.g. 
```
.cupertino-pane-wrapper .inversed .darkmode {

}
``` 
And create styles at beginning. 

Feel free to create pull, i will review and adjust if needs.
And please write here or create pull with some css variables you are think good to use.

**Comment by spb-web** [comment]
If we use classes and “style” tags for styling, then we will have to add the removal of unused “style” tags when the panel is destroyed. It's easier for me to use inline styles. Please see an [example of this](https://github.com/spb-web/cupertino-pane/blob/refactor-styles/src/cupertino-pane.ts#L488). This is not complete code yet, but it is sufficient.

**Comment by Vishal-Isharani** [comment]
@roman-rr @spb-web I'm also facing a wired style issue, where panes styles working weirdly ( no animation or wrong opening positions ).
I have three panes on a single page, two opens from the bottom and one opens from the top ( using inverse mode ).
Ex:- 
Pane a: - Open from top
Pane b: Open from bottom
Pane c: Open from bottom.

Now when you open Pane a first and then go for pane b, pane b will work weirdly, and same if you do reverse. But when you open Pane b and pane c in sequence all works fine.

After hours of debugging I found that, Cupertino creates  style tags for panes and add them to the head section (which is causing issues here), and when you destroy a pane styles stays there it doesn't get removed.
So for fixing the issues I need to remove them manually. I achieved that by doing the following.

```
/**
     * close a pane
     * close a pane and removes it from paneMapper holder
     * @param {string} el
     */
    public close(el: string): void {
        // get pane
        const pane = this.getInstance(el);
        if (pane) {
            // delete from pane mapper
            this.paneMapper.delete(el);
            // destroy pane
            pane.destroy({ animate: true });

            // let animation happen, than remove styles  
            setTimeout(() => {
                this.removeStyles();
            }, this._defaultSettings.animationDuration);
        }
    }
    
private removeStyles(): void {
        const styles = document.getElementsByTagName('style');
        for (let i = 0; i < styles.length; i++) {
            const style = styles.item(i);
            const isCupertinoPaneStyle = style.innerHTML.trim().startsWith('.cupertino-pane-wrapper');
            if (isCupertinoPaneStyle) {
                style.parentNode.removeChild(style);
            }
        }
    }
```
I think @spb-web is right we can use inline-styles, which will eliminate above mentioned issue.
Thanks

**Comment by roman-rr** [comment]
@Vishal-Isharani @spb-web i will personally work on it on the next 24 hrs.

**Comment by spb-web** [comment]
#134

**Comment by roman-rr** [comment]
@Vishal-Isharani Should be fixed in master branch. But in next few days will be released as `1.2.5` with variables also.

**Comment by Vishal-Isharani** [comment]
> @Vishal-Isharani Should be fixed in master branch. But in next few days will be released as `1.2.5` with variables also.

That's is some quick @roman-rr, 
Thanks.
